test/e2e/upgrade/upgrade.go

```diff
@@ -374,11 +387,14 @@ func clusterUpgrade(f *framework.Framework, c configv1client.Interface, dc dynam
 				return cv.Status.ObservedGeneration >= updated.Generation, nil
 
 			}); err != nil {
+				// We allow extra time on a couple platforms above, if we're over the default we'll flake this test
+				// to allow insight into how often we're hitting this problem and when the issue is fixed.
+				flake := time.Now().Sub(start) > defaultCVOUpdateAckTimeout
```

Comment:
if you set the `cvoAckTimeout` to 4 minutes above, then if it takes 3 minutes this code will never be called because there won't be a timeout error.  However, you should still flake.